catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -99,26 +101,203 @@
              description="Catalog core plugins">
         <feature>jackson</feature>
         <bundle>mvn:ddf.catalog.core/catalog-core-resourcesizeplugin/${project.version}</bundle>
-        <bundle>mvn:ddf.catalog.core/catalog-core-resourcestatusplugin/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-tagsfilterplugin/${project.version}</bundle>
-        <bundle>mvn:ddf.catalog.core/catalog-core-camelcomponent/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-commands/${project.version}</bundle>
-        <bundle>mvn:ddf.catalog.core/catalog-core-solrcommands/${project.version}</bundle>
-        <bundle>mvn:ddf.catalog.core/catalog-core-eventcommands/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-metacardgroomerplugin/${project.version}</bundle>
     </feature>
 
     <feature name="catalog-core-content" version="${project.version}"
              description="Core features for content">
         <bundle>mvn:ddf.catalog.core/catalog-core-contentresourcereader/${project.version}</bundle>
         <bundle>mvn:org.codice.ddf/checksum/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-rest" version="${project.version}">
```

Comment:
a few of these features are missing some dependencies in order to be standalone

Reply:
Features in the root features folder have been fixed to include all of their dependencies.  Catalog and spatial have not been fixed yet.